spec/support/cookie_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Spec
+  module Support
+    module Helpers
+      def cookie_expires_value
+        @cookie_expires_value ||= Gem::Version.new(Rack.release) <= Gem::Version.new('2') ? Time.at(0).gmtime.rfc2822 : Time.at(0).httpdate
+      end
+
+      def last_response_cookies
+        Array(last_response.headers['Set-Cookie']).flat_map { |h| h.split("\n") }.sort
+      end
```

Comment:
If you want to make this more elegant, you could monkey patch `last_response` to get `last_response.cookies` and `last_response.cookies.expires_at` or something like that.

Reply:
Its already defined with Rack 3.0 and its using CGI:Cookie to parse cookies. Unfortunately, it doesn't manage `max-age`

Reply:
Exactly. When using an older version of Rack, define the same method as Rack 3.0 by including a monkey patch in https://github.com/ruby-grape/grape/tree/master/spec/support.

https://github.com/rack/rack/blob/64ad26e3381da2ce1853638a2c4ea241c2ad3729/lib/rack/request.rb#L293